app/tests/e2e/test_proof_request_models.py

```diff
@@ -0,0 +1,121 @@
+import pytest
+from aries_cloudcontroller import (
+    AcaPyClient,
+    V10PresentationSendRequestRequest,
+    V20PresRequestByFormat,
+    V20PresSendRequestRequest,
+)
+from fastapi import HTTPException
+
+from app.models.tenants import CreateTenantResponse
+from app.routes.verifier import router as verifier_router
+from app.services.event_handling.sse_listener import SseListener
+from app.tests.util.ecosystem_connections import AcmeAliceConnect
+from shared.util.rich_async_client import RichAsyncClient
+
+VERIFIER_BASE_PATH = verifier_router.prefix
+
+
+@pytest.mark.anyio
+@pytest.mark.xfail(raises=HTTPException, strict=True)
```

Comment:
I think xfail takes a "reason" argument, which we should populate. Something like: 
"Assert `version` and `name` are required fields, despite being optional according to ACA-Py models".